ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/BlockDataGenerator.java

```diff
@@ -335,7 +335,9 @@ private BlockHeader ommer() {
   }
 
   private TransactionType transactionType() {
-    return transactionType(TransactionType.values());
+    // TODO: when TransactionType.EIP4844 is fully supported, revert this.
```

Comment:
Idk how this work, but just wondering if is it possible to change the return value based on the fact that Cancun is enabled or not

Reply:
it's probably possible but since this is in test-support I don't think it is worth it.

Reply:
fair, the important thing is that we do not forget about it :)